README.md

```diff
@@ -22,6 +22,11 @@ The following steps will ensure your project is cloned properly.
 2. `cd SpongeVanilla`
 3. `cp scripts/pre-commit .git/hooks`
 
+**Note**: If accidentally omit `--recursive` option when cloning, you may delete
+repository and try again, or you may execute following to fix the situation:
```

Comment:
A few words seem to be missing:

* “you” between “if” and “accidentally”
* “the” between “delete” and “repository”
* “the” between “execute” and “following”